app/helpers/categories_helper.rb

```diff
@@ -0,0 +1,19 @@
+module CategoriesHelper
+  def create_category(category)
+    if category.save
```

Comment:
this method is a little deceiving from the categories controller. I was unsure what it was handling...maybe something like `attempt_create_category`